src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -418,7 +418,7 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
 	bool		control_position(const Vector2f &curr_pos, const Vector2f &ground_speed, const position_setpoint_s &pos_sp_prev,
 					 const position_setpoint_s &pos_sp_curr);
 	void		control_takeoff(const Vector2f &curr_pos, const Vector2f &ground_speed, const position_setpoint_s &pos_sp_prev,
-					const position_setpoint_s &pos_sp_curr);
+					const position_setpoint_s &pos_sp_curr, float &pre_takeoff_throttle);
```

Comment:
I might not be following this correctly, but why do we need to pass this through to the takeoff routine?

Reply:
Good point. I'm not sure what my original thought process was there. -- will change it and just call the throttle idle get from launch detection whenever it's needed.